tests/sentry/models/test_project.py

```diff
@@ -407,6 +409,55 @@ def test_remove_team_clears_alerts(self):
         assert alert_rule.user_id is None
 
 
+class ProjectSettings(TestCase):
+    def setUp(self):
+        super().setUp()
+        self.option_key = "sentry:test_data"
+        self.project = self.create_project()
+
+        self.project_template = self.create_project_template(organization=self.project.organization)
+        self.project.template = self.project_template
+
+    def tearDown(self):
+        super().tearDown()
+
+        self.project_template.delete()
+        self.project.delete()
+
+    def test_get_option(self):
+        assert self.project.get_option(self.option_key) is None
+        ProjectOption.objects.set_value(self.project, self.option_key, True)
+        assert self.project.get_option(self.option_key) is True
+
+    def test_get_template_option(self):
+        assert self.project.get_option(self.option_key) is None
+        ProjectTemplateOption.objects.set_value(self.project_template, self.option_key, "test")
+        assert self.project.get_option(self.option_key) == "test"
+
+    def test_get_option__overide_template(self):
```

Comment:
nittiest nit of all nits but are these test names meant to have double underscores?

Reply:
Also override with 2 r's

Reply:
yeah - the double underscores are used to split apart the "name" of the test with the "context" of the test. so in this case i'm "testing" the "get_option" function and the context is that i want to "override the template".

idk - something i've done in the past in python to help organize all the tests, but happy to remove them if it's too confusing.

---

will fix the 2 r's - and curious about your opinion on ^ for organizing tests.

Reply:
ah I see - that makes sense, I just haven't seen tests named like that before.